### ISSUE TITLE
debug multiple classes showing up on binary synthetic data - facilitates correct mask visualizations representative of the number of classes

### ISSUE THREAD

**Issue by yalaudah** [microsoft/seismic-deeplearning#326]
Investigate why the model predictions in the binary dataset result in images with multiple classes like this image:
![image](https://user-images.githubusercontent.com/11019911/83083680-5b6c3c00-a04c-11ea-883c-16ba0dce3e88.png)
![image](https://user-images.githubusercontent.com/11019911/83083874-efd69e80-a04c-11ea-96ea-9d71c2fa1ccf.png)

This might be due to the logits being passed on to `mask_to_disk()` before taking the max()

**Comment by yalaudah** [comment]
I remove the other issue (binary image becomes non binary after augmentation) since it is unrelated. It is not referenced in issue: #344
